fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1544,34 +1544,83 @@ Future<Void> deleteTenant(Reference<DB> db, int64_t id) {
 	return Void();
 }
 
-ACTOR template <class Transaction>
-Future<std::vector<std::pair<TenantName, TenantMapEntry>>> listTenantsTransaction(Transaction tr,
-                                                                                  TenantNameRef begin,
-                                                                                  TenantNameRef end,
-                                                                                  int limit) {
-	tr->setOption(FDBTransactionOptions::RAW_ACCESS);
```

Comment:
You can leave the `RAW_ACCESS` option on these functions because that helps enforce that we aren't calling this function from a tenant transaction.